src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -213,9 +213,9 @@ def default_api_version(self):
         ResourceType.MGMT_ARO: '2020-04-30',
         ResourceType.MGMT_DATABOXEDGE: '2019-08-01',
         ResourceType.MGMT_CUSTOMLOCATION: '2021-03-15-preview',
-        ResourceType.MGMT_CONTAINERSERVICE: SDKProfile('2021-03-01', {
+        ResourceType.MGMT_CONTAINERSERVICE: SDKProfile('2021-05-01', {
             'container_services': '2017-07-01',
-            'open_shift_managed_clusters': '2019-10-27-preview'
+            'open_shift_managed_clusters': '2019-09-30-preview'
```

Comment:
The profile used here is introduced in this [PR#18118](https://github.com/Azure/azure-cli/pull/18118). But actually open shift commands depend on the operations provided by the SDK with api version "2019-09-30-preview". May refer [this file](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/acs/_client_factory.py#L68).